src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,264 @@
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement
+                            = connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            if (resultSet.next()) {
+                car = getCar(resultSet);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get car by id  " + id, e);
+        }
+        if (car != null) {
+            car.setDrivers(getDrivers(id));
+        }
+        return car;
+    }
+
+    @Override
+    public List<Car> getAll() throws SQLException {
+        Car car = null;
+        List<Car> cars = new ArrayList<>();
+        List<Driver> drivers = new ArrayList<>();
+        String query = "SELECT taxy_service_cars.id, "
+                + "taxy_service_cars.model, taxy_service_cars.manufacturer_id, "
+                + " manufacturers.name, manufacturers.country, "
+                + "drivers.id, drivers.name, drivers.license_number "
+                + "FROM taxy_service.taxy_service_cars "
+                + "JOIN manufacturers "
+                + "ON manufacturers.id = manufacturer_id "
+                + "JOIN cars_drivers "
+                + "ON car_id = taxy_service_cars.id "
+                + "JOIN drivers "
+                + "ON driver_id = drivers.id "
+                + "order by taxy_service_cars.id;";
+        ResultSet resultSet = null;
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                car = getCar(resultSet);
+                cars.add(car);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get all cars", e);
+        }
+        for (Car c : cars) {
+            c.setDrivers(getDrivers(c.getId()));
+        }
+        return cars;
+    }
+
+    @Override
+    public Car update(Car car) {
+        String query = "update taxy_service_cars set model = ?, manufacturer_id = ? "
+                + "where taxy_service_cars.id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setString(1, car.getModel());
+            statement.setLong(2,car.getManufacturer().getId());
+            statement.setLong(3,car.getId());
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't udate car by id " + car.getId(), e);
+        }
+        for (Driver d: car.getDrivers()) {
+            carService.removeDriverFromCar(d,car);
+        }
+        for (Driver d: car.getDrivers()) {
+            carService.addDriverToCar(d,car);
+        }
```

Comment:
You can't use service class in dao layer. Create separate private methods to remove and add relations between car and driver